src/test/resources/retest/recheck/de.retest.web.PageFrameIT/page-frame-FirefoxDriver.open.recheck/retest.xml

```diff
@@ -7137,7 +7137,7 @@
 				</containedComponents>
 			</containedComponents>
 			<screenshot>
-				<persistenceId>window_2be85b78d7369922356b694c2315e1ffaedd00d59f56aeb3437e079dec3cc527</persistenceId>
+				<persistenceId>window_cc02e51500a14058e890ac79be4884ec8e05c46394e148d42dea5afbc2e5cfcd</persistenceId>
```

Comment:
Since now we don't have that many different screenshots anymore, we probably should use a better (i.e. fixed) name. How about `window_1.png` and `window_2.png` ...? This is probably a change in retest-core, though.